packages/cozy-sharing/locales/en.json

```diff
@@ -381,8 +381,8 @@
       }
     },
     "fingerprint": {
-      "instruction": "Every Cozy is associated to a unique **fingerprint phrase**. For security reasons, please ask to %{userName} (%{userEmail}) if his Cozy's fingerprint phrase is corresponding to :",
-      "instruction2": "The fingerprint phrase of a Cozy is visible on top of the Cozy Pass' passwords list. This fingerprint phrase can also be accessed on **Parameters/Profile/Fingerprint phrase** from the mobile client or the browser extension."
+      "instruction": "For security reasons, please verify **%{userName} (%{userEmail})** identity. To do this, contact **%{userName}** using means of communcation of your choice (call, text message, instant messaging app, ...) and verify his security code.<br><br>Every Cozy user has a security code that he can find using his web application. His security code must correspond to :",
```

Comment:
- I would add a possessive between `userEmail` and `identity`
- I would say `via the means` instead of `using means`
- Typo : `communication`
- No comma before an ellipse
- The neutral form in English is third person plural (they/their/theirs)

Reply:
Thx, I did the fixes. Here is the new result

![image](https://user-images.githubusercontent.com/1884255/132724427-f182e12b-57bb-4fd4-b230-ea68b0d7deba.png)

Reply:
@Ldoppea you missed a `his` to transform into a `their`

Reply:
Good catch.

New version with
- his replaced by their
- space removed before colons in en.json
- space replace by non-breaking space in fr.json